### DISCUSSION TITLE
pretrained checkpoints details

### DISCUSSION THREAD

**Discussion by minouei-kl** [mindee/doctr#839] [Q&A] [answered]
Great work! 
I couldn't find any information about the available pretrained checkpoints. 
I like to know which datasets are used for training them and how many epochs they were trained.

**Comment by fg-mindee** [comment, marked as answer]
Hi @minouei-kl :wave: 

Thanks for the question! This suggests we haven't done our documentation duty well enough :sweat_smile: 

About test sets, you can find all the information over here :point_right: https://mindee.github.io/doctr/latest/using_models.html
About training sets, the answer is more complex! Generally speaking, you can always refer to the [reference training scripts](https://github.com/mindee/doctr/tree/main/references). More specifically for each task:
- character classification : we use a fully synthetic dataset that is home-made and available https://mindee.github.io/doctr/latest/datasets.html#doctr.datasets.CharacterGenerator
- text detection : we use a private dataset of about 100k pages of various document types (cannot be shared since it has sensitive information)
- text recognition : also a private dataset of about 10M word crops (sensitive information as well). However, we started experimenting with a synthetic one, not stable yet though (https://mindee.github.io/doctr/latest/datasets.html#doctr.datasets.WordGenerator)
- artefact detection : we also use a fully synthetic home-made dataset https://mindee.github.io/doctr/latest/datasets.html#doctr.datasets.DocArtefacts

Regarding the hyperparameters of the training, we will start documenting that more thoroughly! Let me know if you have any other questions :)

**Reply by ladanisavan** [reply]
Hi @fg-mindee 

I was wondering if you guys have documented hyperparameters used for training the db_resnet50 model. thanks

**Reply by felixdittrich92** [reply]
@charlesmindee @frgfm
